tests/src/test/kotlin/com/android/declarative/tests/LargeDeclarativeTest.kt

```diff
@@ -202,11 +202,13 @@ class LargeDeclarativeTest {
             }.toString()
         )
 
-      File(project.projectDir, "gradle.properties").writeText(
-        StringBuilder().also { builder ->
-          builder.append("""org.gradle.jvmargs=-Xmx6096m""")
-        }.toString()
-      )
+        File(project.projectDir, "gradle.properties").writeText(
+            StringBuilder().also { builder ->
```

Comment:
nit: Not sure why we need a builder here instead of String.

Reply:
true, removed.